shared/set_configs/EMN.json

```diff
@@ -14,11 +15,87 @@
     ]
   },
   "SET_CORRECTIONS": [{
-    "match": {
-      "multiverseid": 414497
+      "match": {
+        "imageName": "brisela, voice of nightmares1"
+      },
+      "replace": {
+        "number": "15b",
+        "names": [
+          "Bruna, the Fading Light",
+          "Brisela, Voice of Nightmares"
```

Comment:
I do not agree with this change. The back face of a meld card is one card, not two. The fact that it is physically separated into two halves is irrelevant. By that logic, we should also combine split, flip, and double-faced cards into one object.

See #187 for prior discussion.